cluster/aws/templates/create-dynamic-salt-files.sh

```diff
@@ -82,6 +82,27 @@ contexts:
 current-context: service-account-context
 EOF
 
+mkdir -p /srv/salt-overlay/salt/kubelet
+kubelet_kubeconfig_file="/srv/salt-overlay/salt/kubelet/kubeconfig"
+cat > "${kubelet_kubeconfig_file}" <<EOF
+apiVersion: v1
+kind: Config
+users:
+- name: kubelet
+  user:
+    token: ${kubelet_token}}
+clusters:
+- name: local
+  cluster:
+     insecure-skip-tls-verify: true
```

Comment:
I think that in AWS the certs are still generated on the master, which means that the kubelet doesn't have the CA cert when it starts up.